contracts/utils/Strings.sol

```diff
@@ -64,4 +64,11 @@ library Strings {
         require(value == 0, "Strings: hex length insufficient");
         return string(buffer);
     }
+
+    /**
+     * @dev Converts an `address` to its ASCII `string` hexadecimal representation.
+     */
+    function addressToHexString(address addr) internal pure returns (string memory) {
+        return toHexString(uint256(uint160(addr)));
```

Comment:
We have to specify a fixed length in case the address starts with zeroes.

Reply:
You are absolutely right! Fixed it here (including test case): https://github.com/OpenZeppelin/openzeppelin-contracts/pull/3403/commits/ac902373e13ad6118c20a905a70e87a174e43111